content/10.diagnostics.md

```diff
@@ -213,7 +213,7 @@ It also reported 100% specificity and sensitivity on 114 RNA samples from clinic
 A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
 They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].
 
-LFT strips are simple to use and read, but there are limitations in terms of availability and cost per test. <!-- Is SHERLOCK more available and less expensive than LFTs? This sentence feels out of place, or I'm missing the point. -->
```

Comment:
This sentence ended up in the wrong paragraph when a paragraph break was added. I think it's a lot more obvious now what it is supposed to mean (but let me know if that's incorrect!)